sync/class.jetpack-sync-defaults.php

```diff
@@ -213,6 +213,8 @@ public static function get_callable_whitelist() {
 		'idx_page',
 		'postman_sent_mail',
 		'rssmi_feed_item',
+		'rssap-feed',
+		'wp_automatic'
```

Comment:
Could you add a trailing comma to avoid issues later on when adding new items, if one were to forget to add that comma? This will also make WPCS happy :)

Thanks!